model/category/category.go

```diff
@@ -166,7 +172,8 @@ func (c Category) IsFromES() bool {
 
 // IsFromRS checks whether the category is of the reactivesearch category.
 func (c Category) IsFromRS() bool {
-	return c == ReactiveSearch
+	return c == ReactiveSearch ||
+	       c == FuzzinessAnalysis
```

Comment:
@bsrk Why is that change required?
`FuzzinessAnalysis` is a new category.

You also need to add the limits for that new category.

Reply:
This plugin does not use olivere's elastic search, but rather uses MakeESRequest in querytranslate.

I'll check and add limits.

Reply:
I added the limits.

Reply:
I removed FuzzinessAnalysis from IsFromRS check, though I anticipate this will allow unauthenticated FuzzinessAnalysis request to pass through. Let's resolve this issue if it actually pops ups.